pkg/cachedmap/cachedmap.go

```diff
@@ -31,6 +31,7 @@ type CachedMap[Key comparable, T any] interface {
 	Clear()
 	Add(key Key, obj *T)
 	Remove(key Key)
+	Keys() []Key
 	ToSlice() []*T
```

Comment:
Name it `Values` then? (It'll be similar to the Keys and Values methods from the exp package https://pkg.go.dev/golang.org/x/exp/maps#Values)